braph2genesis/graphs/_MultilayerWU.gen.m

```diff
@@ -0,0 +1,514 @@
+%% ¡header!
+MultilayerWU < Graph (g, multilayer weighted undirected graph) is a multilayer weighted undirected graph.
+
+%%% ¡description!
+In a multilayer weighted undirected (WU) graph, layers have different number 
+ of nodes with within-layer weighted undirected edges, associated with a real 
+ number between 0 and 1 and indicating the strength of the connection.
+The connectivity matrices are symmetric.
+There are connections between layers connecting the corresponding nodes.
```

Comment:
you need to update your branch based on this one: https://github.com/giovannivolpe/BRAPH-2-Matlab-beta/pull/9
Descriptions are wrong, you have not updated them